cpu/cc26x0/include/cc26x0_rfc.h

```diff
@@ -392,18 +406,28 @@ typedef struct {
 /**@}*/
 
 
-typedef struct {
-    radio_op_command_t op;
-    uint8_t mode;
+typedef struct __attribute__ ((aligned(4))) {
+    _radio_op_command_t op;
+    uint8_t mode; /* W */
     /*
      * supported values: 0 (equivalent to 2), 2, 5, 6, 10, 12, 15, and 30
      * 0 or 2 only supported for devices that support 2.4-GHz operation
      */
-    uint8_t loDivider;
-    uint16_t config;
-    uint16_t txPower;
-    uint32_t pRegOverride;
-} radio_setup_t;
+    uint8_t IoDivider; /* W */
+    struct {
+        uint16_t frontEndMode:3;
+        uint16_t biasMode:1;
+        uint16_t :6;
+        uint16_t bNoFsPowerUp:1;
+    } config; /* W */
+    struct {
+        uint16_t IB:6;
+        uint16_t GC:2;
+        uint16_t boost:1; // TODO not in datashiet?
```

Comment:
@fvcoen tempCoeff is 8 bits long and there is no boost in the datasheet on (page 1492)
how did you find this?